integration-tests/suites/config/config.go

```diff
@@ -5,7 +5,6 @@ const (
 	CollectionMethodCoreBPF = "core-bpf"
 
 	runtimeDefaultCommand = "docker"
-	runtimeDefaultSocket  = "/var/run/docker.sock"
```

Comment:
I think this is messing with this other line.
https://github.com/stackrox/collector/blob/485640ef687aeb3064ea5722fb3ddac6c093438b/integration-tests/suites/config/config.go#L131

You probably want to add it back in.

Reply:
And the one in integration-tests/suites/config/env.go

Reply:
Argh, I missed that usage of the config object. I restored it and the CI now seems happy.